src/cic/core.clj

```diff
@@ -121,5 +149,50 @@
 
 (defn csv->episodes
   [filename]
-  (-> (load-csv filename)
-      (episodes)))
+  (->> (load-csv filename)
+       (map format-episode)
+       (episodes)))
+
+(defn load-duration-csv
```

Comment:
I think all the csv stuff is going to want to go into an input ns subtree with some common things at the input level and then each "domain type" having its own namespace, but I think this is OK for now.